docs/source/user-manual/glossary.rst

```diff
@@ -0,0 +1,45 @@
+Glossary
+========
+
+Becoming familiar with new tools can be daunting.
+In this glossary you will find the most commonly used terms in OpenKAT with it's definition.
+We hope this helps to make your life easier.
+
+
+.. list-table:: Glossary
+   :widths: 15 85
+   :header-rows: 1
+
+   * - Term
+     - Definition
+   * - Object/OOI
+     - The information collected by OpenKAT is stored as objects.
+       Objects can be anything, like DNS records, hostnames, URLs, IP addresses, software, software versions, ports, etc.
+   * - Object type
+     - The type of object, for example IP address, URL, website.
+   * - Finding
+     - Information about the system found by OpenKAT.
+   * - Finding type
+     - The type of information that has been found by OpenKAT.
+   * - Plugin
+     - Deployed by OpenKAT to collect information, translate it into objects for the data model and then analyze it.
+   * - Boefje
+     - A type of plugin, which gathers facts from the objects.
+   * - Task
+     - A task is created for each job that needs to be performed, such as running a plugin or for generating a report.
+       Not every task results in findings, but almost every task results in new objects.
```

Comment:
It depends on the task, some tasks for example only return vulnerabilities and those almost never return new objects.

```suggestion
       Not every task results in findings or new objects.
```